todo_rxbloc_app/lib/feature_todo_details/blocs/todo_details_bloc.dart

```diff
@@ -0,0 +1,64 @@
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rxdart/rxdart.dart';
+import 'package:todos_repository_core/todos_repository_core.dart';
+
+import '../../base/common_blocs/coordinator_bloc.dart';
+import '../../base/models/navigation_parametars.dart';
+import '../../feature_homepage/bloc/navigation_bloc.dart';
+
+part 'todo_details_bloc.rxb.g.dart';
+
+/// A contract class containing all events of the TodoDetailsBloC.
+abstract class TodoDetailsBlocEvents {
+  void toggleTodo(TodoEntity todoEntity, bool isComplete);
+  void navigate(NavigationParams navigationParametars);
+  void deleteTodo(TodoEntity todoEntity);
+}
+
+/// A contract class containing all states of the TodoDetailsBloC.
+abstract class TodoDetailsBlocStates {
+  Stream<TodoEntity> get todoEntity;
+  ConnectableStream<void> get setNavigation;
+  ConnectableStream<void> get todoDeleted;
+}
+
+@RxBloc()
+class TodoDetailsBloc extends $TodoDetailsBloc {
```

Comment:
Hmmm it looks like this BloC does not contain any business logic, instead just sends events to the coordinator bloc and calls function of the navigatorBloc. Can you please elaborate what's the purpose of this bloc ?